templates/store-gateway/store-gateway-statefulset.yaml

```
@@ -16,6 +16,7 @@ spec:
       {{- include "cortex.storeGatewaySelectorLabels" . | nindent 6 }}
   updateStrategy:
     {{- toYaml .Values.store_gateway.strategy | nindent 4 }}
+  podManagementPolicy: "{{ .Values.store_gateway.statefulSet.podManagementPolicy }}"
```

```suggestion
  podManagementPolicy: {{ .Values.store_gateway.statefulSet.podManagementPolicy }}
```

drop the quotes please

---

meh I guess we are not really consistent. I would prefer if you do wanna force quotes to use the helm `| quote` function. I'm good with either change. We might need to tackle this another day

---

will use ` | quote` 👍🏽